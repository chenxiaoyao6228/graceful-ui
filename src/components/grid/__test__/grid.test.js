import { render } from '@testing-library/vue';
import Col from '../col.vue';
import Row from '../row.vue';

describe('Grid', () => {
  it('match Snapshot', () => {
    const testComponent = {
      components: { Row, Col },
      template: `
        <Row :gutter="4">
          <Col span="8" offset="2">
            <div>这里是内容1</div>
          </Col>
          <Col span="8" offset="2">
            <div>这里是内容2</div>
          </Col>
        </Row>
      `
    };
    const { container } = render(testComponent);
    expect(container).toMatchSnapshot();
  });
});
