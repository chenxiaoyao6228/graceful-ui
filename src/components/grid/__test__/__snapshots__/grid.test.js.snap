// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid match Snapshot 1`] = `
<div>
  <div
    class="g-row"
    style="margin-left: -2px; margin-right: -2px;"
  >
    <div
      class="g-col g-col-span-8 g-col-offset-2"
      style="padding-left: 0px; padding-right: 0px;"
    >
      <div>
        这里是内容1
      </div>
    </div>
     
    <div
      class="g-col g-col-span-8 g-col-offset-2"
      style="padding-left: 0px; padding-right: 0px;"
    >
      <div>
        这里是内容2
      </div>
    </div>
  </div>
</div>
`;
